doc/_ext/nope/blending/fade.py

```diff
@@ -0,0 +1,32 @@
+from pynopegl_utils.misc import SceneCfg, scene
+
+import pynopegl as ngl
+
+
+@scene()
+def fade(cfg: SceneCfg):
+    jpegs = (m for m in cfg.medias if m.filename.endswith(".jpg"))
```

Comment:
`jpegs` -> `images`, `.jpg` + `.png` (eventually webp)

Reply:
I actually just want the jpegs here because I want some background picture and not an icon